server/src/test/java/io/spine/server/outbus/enrich/EnricherBuilderTest.java

```diff
@@ -51,15 +52,15 @@
 class EnricherBuilderTest {
 
     private Enricher.AbstractBuilder builder;
-    private Function<Timestamp, StringValue> function;
+    private BiFunction<Timestamp, Int32Value, StringValue> function;
     private FieldEnrichment<Timestamp, StringValue, ?> fieldEnrichment;
 
     @BeforeEach
     void setUp() {
         this.builder = EventEnricher.newBuilder();
-        this.function = new Function<Timestamp, StringValue>() {
+        this.function = new BiFunction<Timestamp, Int32Value, StringValue>() {
```

Comment:
Please use the value second parameter type (e.g. `EventContext`) here and below.

Once we get rid of rejections (and they become rejection events), the second parameter will always be `EventContext`; that includes `C` parameter in the `FieldEnrichment` class, which will be removed from the generic params at all.